src/lib/actions.ts

```
@@ -0,0 +1,92 @@
+  const handle = options.handle_selector
+    ? node.querySelector<HTMLElement>(options.handle_selector)
+    : node
+
+  if (!handle) {
+    console.warn(`Draggable: handle not found with selector "${options.handle_selector}"`)
+    return { destroy: () => {} }
+  }
+
+  function handle_mousedown(event: MouseEvent) {
+    // Only drag if mousedown is on the handle or its children
+    if (handle && handle.contains && !handle.contains(event.target as Node)) return
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)